src/status_im/common/bottom_sheet/view.cljs

```diff
@@ -62,24 +62,33 @@
            gradient-cover? customization-color hide-handle? blur-radius]
     :or   {border-radius 12}}]
   (let [theme                             (quo.theme/use-theme-value)
-        sheet-height                      (rn/use-ref-atom 0)
-        item-height                       (rn/use-ref-atom 0)
+        [sheet-height set-sheet-height]   (rn/use-state 0)
+        handle-sheet-height               (rn/use-callback (fn [e]
+                                                             (when (= sheet-height 0)
+                                                               (set-sheet-height
+                                                                (.-nativeEvent.layout.height ^js e)))))
```

Comment:
@flexsurfer we should be using state here (and below), otherwise it won't re-render the `selected-item` will not be visible.

Reply:
but then you need to pass it as dep for `use-callback`